assets/src/blocks/Happypoint/HappypointBlock.js

```diff
@@ -0,0 +1,62 @@
+import {HappypointEditor} from './HappypointEditor';
+import {HappyPointV1} from './deprecated/HappyPointV1.js';
```

Comment:
Maybe we should be consistent with the name case? It seems we use `Happypoint` everywhere else